Mono.Debugging.Soft/Mono.Debugging.Soft.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
```

Comment:
We're working towards slowly phasing out net472, because I think that implies a dependency on Mono, so I wonder if this should conditionally target net472 and net6.0.

Reply:
Do you want me to add some property that could control multitargeting this in this current PR, or is this something we should do in a future PR?

Both [Mono.Debugger.Soft](https://github.com/mono/debugger-libs/blob/eaf83aacd4f5b41e240ceb3a61dd49f64227370f/Mono.Debugger.Soft/Mono.Debugger.Soft.csproj#L4) and [Mono.Debugging](https://github.com/mono/debugger-libs/blob/eaf83aacd4f5b41e240ceb3a61dd49f64227370f/Mono.Debugging/Mono.Debugging.csproj#L4) are targeting `net6.0` and `net472`, so I figured this removal of `net472` was maybe just an oversight.

Reply:
I'll deal with it if it becomes a problem then! I saw that `net472` shells out to `mono` when building, so when we remove `mono`, if it breaks, I'll do that.